.astylerc

```diff
@@ -18,3 +18,4 @@
 --pad-oper
 --suffix=none
 --unpad-paren
+
```

Comment:
Do you think a trailing newline is appropriate here, or was it a mistake?

Reply:
Mistake - and I'm not sure how I managed it.  Let me remove...